### ISSUE TITLE
Bump version of www-client/min

### ISSUE THREAD

**Issue by tajo48** [BlueManCZ/edgets#17]
1.29.0 came out so perhaps its worth updating this ebuild
Its worth upgrading especially because 1.27 is packed with high score electron/chromium CVEs

**Comment by BlueManCZ** [comment]
Sure thing, thank you for pointing this out. I updated the ebuild, but it doesn't work correctly for me. It may be a problem related to my specific setup (using 525.125.06 Nvidia drivers and Wayland), but Min crashes after a few seconds with stdout full of errors:

```
MESA-LOADER: failed to retrieve device information
MESA-LOADER: failed to open nvidia-drm: /usr/lib64/dri/nvidia-drm_dri.so: cannot open shared object file: Permission denied (search paths /usr/lib64/dri, suffix _dri)
MESA-LOADER: failed to open zink: /usr/lib64/dri/zink_dri.so: cannot open shared object file: Permission denied (search paths /usr/lib64/dri, suffix _dri)
MESA-LOADER: failed to open kms_swrast: /usr/lib64/dri/kms_swrast_dri.so: cannot open shared object file: Permission denied (search paths /usr/lib64/dri, suffix _dri)
MESA-LOADER: failed to open swrast: /usr/lib64/dri/swrast_dri.so: cannot open shared object file: Permission denied (search paths /usr/lib64/dri, suffix _dri)
[7855:0914/183726.192639:ERROR:nss_util.cc(357)] After loading Root Certs, loaded==false: NSS error code: -8018
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at sendIPCToWindow (/opt/Min/resources/app/main.build.js:943:24)
    at Function.<anonymous> (/opt/Min/resources/app/main.build.js:2160:7)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:525:35)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:525:35)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:525:35)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1718:31)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/opt/Min/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at sendIPCToWindow (/opt/Min/resources/app/main.build.js:943:24)
    at BrowserWindow.<anonymous> (/opt/Min/resources/app/main.build.js:1101:7)
    at BrowserWindow.emit (node:events:525:35)
```
Let me know if it works fine for you.

**Comment by tajo48** [comment]
> libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at WebContents.<anonymous> (/tmp/.mount_Min-1.NLzg2r/resources/app/main.build.js:1790:29)
    at WebContents.emit (node:events:513:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:88963)
    at WebContents.emit (node:events:513:28)
^C%

> 

Same thing on Arch & Gentoo when building by hand with drm modset set to 1 and X11
Its probably just min's issue
I will check for possible solutions

**Comment by tajo48** [comment]
https://github.com/minbrowser/min/issues/2321

**Comment by BlueManCZ** [comment]
The rendering issue was resolved in 1.30.0.
